src/main/java/zeenea/connector/visualization/Visualization.java

```diff
@@ -136,6 +148,17 @@ public Builder linkedDataset(@NotNull List<ItemReference> linkedDataset) {
       return this;
     }
 
+    /**
+     * Sets the linked dataset references of the visualization.
+     *
+     * @param linkedDataset the linked dataset references of the visualization
+     * @return the Builder instance
+     */
+    public Builder linkedDataset(ItemReference... linkedDataset) {
```

Comment:
On en a plusieurs, donc pluriel, non ? (à voir aussi sur la méthode juste au-dessus) : 
```suggestion
    public Builder linkedDatasets(ItemReference... linkedDataset) {
```

Reply:
En effet, j'ai fait une passe sur toutes les classes pour uniformiser la nomenclature des listes (plus de XXXList, on partage les mêmes noms entre les classes lorsqu'ils représentent la même chose linkedDatasets => sourceDatasets comme dans Dataset).

Je te laisse me dire si cela te convient ;) (merci pour la revue)